phoenicis-scripts/src/main/java/org/phoenicis/scripts/engine/JSAScriptEngine.java

```diff
@@ -0,0 +1,66 @@
+package org.phoenicis.scripts.engine;
+
+import javax.script.ScriptEngine;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Consumer;
+
+public class JSAScriptEngine implements PhoenicisScriptEngine {
```

Comment:
What is "JSA"?

Reply:
JSA = Java Scripting API
See also https://docs.oracle.com/javase/8/docs/technotes/guides/scripting/prog_guide/api.html

Reply:
Ok, let me put it differently: Is the abbreviation common enough to know what this class is about (it wasn't for me at least)? At minimum, we should have a comment.

Reply:
Done